Source/WebCore/platform/audio/AudioArray.h

```diff
@@ -67,38 +65,20 @@ class AudioArray {
         // optimized implementation if possible, which makes the result deterministic.
         constexpr size_t alignment = 32;
 
-        fastAlignedFree(m_allocation);
-
-        m_allocation = static_cast<T*>(fastAlignedMalloc(alignment, initialSize));
-        if (!m_allocation)
-            CRASH();
-        m_size = n;
+        fastAlignedFree(m_allocation.data());
+        m_allocation = unsafeMakeSpan(static_cast<T*>(fastAlignedMalloc(alignment, initialSize)), n);
```

Comment:
If we gave MallocSpan an alignedMalloc() function, we could drop the unsafeMakeSpan.

Reply:
The tricky part is that `fastAlignedMalloc()` needs to be paired with `fastAlignedFree()` it seems.

Reply:
I'll create a FastAlignedMalloc struct and use that instead of FastMalloc I think. Then we don't need to add an `alignedMalloc()` function to `MallocSpan` because `malloc()` would do the right thing.

Reply:
Actually, we'll still need an explicit `alignedMalloc()` function because of the extra alignment parameter.

Reply:
FWIW, alignedFree() was only required by old versions of Windows, so we could potentially get rid of it at some point. But this is great as is.